oxo.py

```diff
@@ -1,25 +1,49 @@
+
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+
+        ##This is the code for the array board and its spaces
+        self.theboard = [[0,0,0][0,0,0][0,0,0]]
+        self.col = len(self.board)
+        ##Responsible for the number of rows based on the board
+        self.rownumber = len(self.board[0])
 
     def get_square(self, x, y):
-        """ Return 0, 1 or 2 depending on the contents of the specified square. """
+        """ Return the board's state based on the x and y. """
+        return self.theboard[x][y]
         raise NotImplementedError("TODO: implement get_square")
```

Comment:
Don't forget to remove the `raise NotImplementedError` lines as you implement the functions, otherwise they may cause your code to crash